include/packager/live_packager_export.h

```diff
@@ -94,30 +94,36 @@ size_t livepackager_buf_size(LivePackagerBuffer_t buf);
 
 typedef struct LivePackager_instance_s* LivePackager_t;
 
+typedef struct LivePackagerStatus_s* LivePackagerStatus_t;
```

Comment:
We can access the struct directly in CGO,  do we need to define a pointer type for it (and have to deal with memory management)?